server/data/hmppsManageUsersClient.ts

```diff
@@ -0,0 +1,22 @@
+import RestClient from './restClient'
+import logger from '../../logger'
+import config from '../config'
+import type { User } from '@accredited-programmes/users'
+
+export default class HmppsManageUsersClient {
+  restClient: RestClient
+
+  constructor(token: Express.User['token']) {
+    this.restClient = new RestClient('HMPPS Manage Users Client', config.apis.hmppsManageUsers, token)
+  }
+
+  getCurrentUsername(): Promise<Pick<User, 'username'>> {
+    logger.info(`Getting username: calling HMPPS Manage Users`)
+    return this.restClient.get({ path: '/users/me' }) as Promise<Pick<User, 'username'>>
+  }
+
+  getUserFromUsername(username: string): Promise<User> {
```

Comment:
or `findUserFromUsername`

Reply:
I've left it as `getUserFromUsername` for now.  We have `getCurrentUsername` in there too but I'm not sure that works with `find`, so to have a mix of `find` and `get` feels a bit confusing.